src/common_backend.py

```diff
@@ -0,0 +1,47 @@
+"""
+Common methods and functions for kivy and qt.
+"""
+
+import queues
+from bmconfigparser import BMConfigParser
+from bitmessagekivy.baseclass.common import toast
+
+
+class AddressGenerator(object):
+    """"Base class for address generation and validation"""
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def generate_address(entered_label):
+        """"Return True if the label is uniqe"""
+        streamNumberForAddress = 1
+        eighteenByteRipe = False
+        nonceTrialsPerByte = 1000
+        payloadLengthExtraBytes = 1000
+        labels = [BMConfigParser().get(obj, 'label')
+                  for obj in BMConfigParser().addresses()]
+        if entered_label and entered_label not in labels:
+            toast('Address Creating...')
```

Comment:
there needs to be some sort of callback, or a return value to handle this, as `toast` is kivy-specific